packages/conf/src/types.ts

```diff
@@ -0,0 +1,77 @@
+import { GeneratorOptions } from "@babel/core"
+import { ExtractOptions } from "@lingui/cli/src/api/extractors"
```

Comment:
This import is incorrect. conf should not rely on cli package, please see dicussion: 

https://discord.com/channels/974702239358783608/1072883644273795082